File: AppMovil/LuckyPets2/app/src/main/java/com/vgt/luckypets/model/TarjetaBancariaDTO.kt
package com.vgt.luckypets.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class TarjetaBancariaDTO(
    @SerializedName("id")
    val id: Long? = null,
    @SerializedName("numeroTarjeta")
    val numeroTarjeta: Long,
    @SerializedName("fechaCaducidad")
    val fechaCaducidad: String,
    @SerializedName("titularTarjeta")
    val titularTarjeta: String,
    @SerializedName("emisorTarjeta")
    val emisorTarjeta: String,
    @SerializedName("cvv")
    val cvv: Int,
    @SerializedName("imgTarjeta")
    val imgTarjeta: String,
    @SerializedName("usuarioEmail")
    val usuarioEmail: String
) : Serializable

File: AppEscritorio/LuckyPets/TarjetaBancariaDTO.cs
using System;

public class TarjetaBancariaDTO
{
    public long Id { get; set; }
    public long NumeroTarjeta { get; set; }
    public DateTime FechaCaducidad { get; set; }
    public string TitularTarjeta { get; set; }
    public string EmisorTarjeta { get; set; }
    public int Cvv { get; set; }
    public string ImgTarjeta { get; set; }

}